redbot/core/utils/antispam.py

```diff
@@ -45,14 +47,16 @@ def __interval_check(self, interval: AntiSpamInterval):
     @property
     def spammy(self):
         """
-        use this to check if any interval criteria are met
+        Whether any antispam intervals are active.
```

Comment:
It might be worth improving this further with the description that you can find in the comment above the `AntiSpam.spammy` usage in the code example I suggested for the class docstring.

Reply:
```suggestion
        Whether, for any interval, the number of events that happened
        within that interval exceeds the number specified for that interval.
```